management-app/src/components/CreateModal/CreateModal.tsx

```diff
@@ -29,11 +30,51 @@ function CreateModal({
   titleModal,
   labelName,
   btnName,
-  isDisabled,
-  onCreate,
+  onSubmit,
   onClose,
-  onChange,
+  isRenderDescription,
+  descriptionName,
+  labelDescription,
 }: TCreateModal) {
+  const [titleInput, setTitleInput] = useState<string>('');
+  const [taskDescription, setTaskDescription] = useState<string>('');
+  const [isDisabled, setDisabled] = useState<boolean>(true);
+
+  const handleInputChange = useCallback(
+    (event: ChangeEvent<HTMLInputElement>) => {
+      const target = event.target as HTMLInputElement;
+      const value = target.value as string;
+
+      setTitleInput(value);
+    },
+    []
+  );
+
+  const handleChangeDescription = useCallback(
```

Comment:
It seems it's no need to use useCallBack() in this case

Reply:
Why?